app/models/yellow_ribbon_program_source.rb

```diff
@@ -4,48 +4,89 @@ class YellowRibbonProgramSource < ApplicationRecord
     'zip' => { column: :zip },
-    'public private' => { column: :public_private },
-    'updated for 2011-2012' => { column: :updated_for_2011_2012, converter: BooleanConverter },
-    'missed deadline' => { column: :missed_deadline, converter: BooleanConverter },
-    'ineligible' => { column: :ineligible, converter: BooleanConverter },
-    'date agreement received' => { column: :date_agreement_received },
-    'dat yr signed by yr official' => { column: :date_yr_signed_by_yr_official, converter: DateConverter },
+
+    # Unused by GIDS but provided in source file
     'amendment date' => { column: :amendment_date, converter: DateConverter },
-    'flight school' => { column: :flight_school, converter: BooleanConverter },
-    'date confirmation sent' => { column: :date_confirmation_sent, converter: DateConverter },
+    'campus' => { column: :campus },
     'consolidated agreement' => { column: :consolidated_agreement, converter: BooleanConverter },
+    'dat yr signed by yr official' => { column: :date_yr_signed_by_yr_official, converter: DateConverter },
+    'date agreement received' => { column: :date_agreement_received },
+    'date confirmation sent' => { column: :date_confirmation_sent, converter: DateConverter },
+    'flight school' => { column: :flight_school, converter: BooleanConverter },
+    'ineligible' => { column: :ineligible, converter: BooleanConverter },
+    'initials yr processor' => { column: :initials_yr_processor },
+    'missed deadline' => { column: :missed_deadline, converter: BooleanConverter },
+    'modified' => { column: :modified, converter: BooleanConverter },
     'new school' => { column: :new_school, converter: BooleanConverter },
+    'notes' => { column: :notes },
     'open ended agreement' => { column: :open_ended_agreement, converter: BooleanConverter },
-    'modified' => { column: :modified, converter: BooleanConverter },
-    'withdrawn' => { column: :withdrawn, converter: BooleanConverter },
+    'public private' => { column: :public_private },
+    'school name in weams' => { column: :school_name_in_weams },
+    'sco email address' => { column: :sco_email_address },
     'sco name' => { column: :sco_name },
     'sco telephone number' => { column: :sco_telephone_number },
-    'sco email address' => { column: :sco_email_address },
+    'sfr email address' => { column: :sfr_email_address },
     'sfr name' => { column: :sfr_name },
     'sfr telephone number' => { column: :sfr_telephone_number },
-    'sfr email address' => { column: :sfr_email_address },
-    'initials yr processor' => { column: :initials_yr_processor },
+    'updated for 2011-2012' => { column: :updated_for_2011_2012, converter: BooleanConverter },
+    'withdrawn' => { column: :withdrawn, converter: BooleanConverter },
     'year of yr participation' => { column: :year_of_yr_participation },
-    'notes' => { column: :notes }
   }.freeze
 
-  validates :facility_code, presence: true
+  # validates :city, presence: true
```

Comment:
I left these commented as their presence, while not needed, would certainly be useful since much of our design spec depends on them.

Reply:
If these fields are always populated in the CSV and are needed for you work i would uncomment these
Additionally you can list multiple fields per validates, see https://guides.rubyonrails.org/active_record_validations.html#presence

Reply:
These are definitely not needed anymore since we're relying on the `yellow_ribbon_program.rb` model now.

Reply:
Validates only run for tables that are populated by CSV import as https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb#L449 uses pure SQL and does not trigger validates on objects

If you would like to validate that these fields have values then `yellow_ribbon_program_source` model needs the validates, however if you are going to ignore rows without these values at https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb#L449 then the validates are not needed